ocaml/xapi/cli_frontend.ml

```diff
@@ -106,6 +106,15 @@ let rec cmdtable_data : (string*cmd_spec) list =
       flags=[Neverforward];
     };
 
+    "enable-getter-log",
```

Comment:
Seconded that this should go in the config file - otherwise the setting will be lost every time xapi is restarted.